### PR TITLE
Fix fee levels when there is no wallet and hide Glidera if wallet is not complete

### PR DESCRIPTION
**Pull request by gabrielbazan7**
fix : #4013

### CODE REVIEW

**Comment by isocolsky** [comment]
I believe this needs https://github.com/bitpay/bitcore-wallet-client/pull/235 to work. You should add 'WIP' until that gets merged.

File: src/js/controllers/preferencesGlobal.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ angular.module('copayApp.controllers').controller('preferencesGlobalController',
       this.usePushNotifications = isCordova && !isMobile.Windows();
       $scope.PNEnabledByUser = true;
       $scope.isIOS = isMobile.iOS();
-      if (!typeof cordova.plugins.diagnostic != undefined && $scope.isIOS) {
+      if ($scope.isIOS) {
```
Comment (gabrielbazan7) [inline src/js/controllers/preferencesGlobal.js]: cordova is not define fixed

File: src/js/services/feeService.js [hunk 2]
```diff
@@ -19,25 +19,32 @@ angular.module('copayApp.services').factory('feeService', function($log, profile
...
-    var fc = profileService.focusedClient;
     var unitName = configService.getSync().wallet.settings.unitName;
 
-    fc.getFeeLevels('livenet', function(errLivenet, levelsLivenet) {
-      fc.getFeeLevels('testnet', function(errTestnet, levelsTestnet) {
+    walletClient.getFeeLevels('livenet', function(errLivenet, levelsLivenet) {
+      walletClient.getFeeLevels('testnet', function(errTestnet, levelsTestnet) {
         if (errLivenet || errTestnet) $log.debug('Could not get dynamic fee');
         else {
           for (var i = 0; i < 3; i++) {
```
Comment (isocolsky) [inline src/js/services/feeService.js]: What if the server returns more than 3 levels?

**Comment by gabrielbazan7** [comment]
Updated

File: src/js/controllers/preferencesGlobal.js (hunk 1 above)
Comment (cmgustavo) [inline src/js/controllers/preferencesGlobal.js]: `isMobile.iOS()` is for web-browsers. Use `isCordova` instead.

File: src/js/controllers/index.js [hunk 3]
```diff
@@ -295,6 +295,8 @@ angular.module('copayApp.controllers').controller('indexController', function($r
   self.updateAll = function(opts, initStatusHash, tries) {
     tries = tries || 0;
     opts = opts || {};
+    var fc = profileService.focusedClient;
+    if (!fc) return;
 
     var walletId = profileService.focusedClient.credentials.walletId
```
Comment (cmgustavo) [inline src/js/controllers/index.js]: Use `fc` defined above.

**Comment by gabrielbazan7** [comment]
Updated

**Comment by gabrielbazan7** [comment]
Updated

**Comment by cmgustavo** [comment]
ACK

**Pull request merged**